packages/migrate/src/types.ts

```diff
@@ -253,6 +254,11 @@ export namespace EngineArgs {
     columnName: string
   }
 
+  type AffectedObject = {
+    type: string
+    name: string
+  }
```

Comment:
A comment at the top of this object type declaration would be nice

Reply:
What kind of comment would you expect here that you would find helpful?

Reply:
Updated to:
```ts
// This represents any database object
// Model, enum, view, column, etc.
type AffectedObject = {
  type: string // Tells us what kind of database object
  name: string
}
```